crates/sui-rpc-benchmark/src/json_rpc/mod.rs

```diff
@@ -16,11 +16,13 @@ pub async fn run_benchmark(
     file_path: &str,
     concurrency: usize,
     duration_secs: u64,
+    methods_to_skip: HashSet<String>,
 ) -> Result<()> {
     let config = BenchmarkConfig {
         concurrency,
         duration: Duration::from_secs(duration_secs),
         json_rpc_file_path: Some(file_path.to_string()),
+        json_rpc_methods_to_skip: methods_to_skip.into_iter().collect(),
```

Comment:
Seems like the types match -- why the `into_iter()` and `collect()`?

```suggestion
    json_rpc_methods_to_skip: HashSet<String>,
) -> Result<()> {
    let config = BenchmarkConfig {
        concurrency,
        duration: Duration::from_secs(duration_secs),
        json_rpc_file_path: Some(file_path.to_string()),
        json_rpc_methods_to_skip,
```